walk-in-interview/src/main/webapp/new-job.html

```diff
@@ -4,22 +4,22 @@
 <head>
     <meta charset="UTF-8">
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <script src="new-job.js"></script>
 </head>
 
 <body>
 
-    <form>
+    <form id="new-job-form" onsubmit="addJob()">
```

Comment:
ah the return of submit. does that mean that your POST API will return html instead of json?